### PR TITLE
Update repo links

### PR DESCRIPTION
**Pull request by richtabor**
This PR updates the repo links to use `/godaddy-wordpress/`, as well as fixes a few minor URLs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)](https://circleci.com/gh/godaddy/coblocks) [![WordPress plugin](https://img.shields.io/wordpress/plugin/dt/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress plugin](https://img.shields.io/wordpress/plugin/v/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress](https://img.shields.io/wordpress/v/coblocks.svg?style=flat)]() [![License](https://img.shields.io/badge/license-GPL--2.0%2B-red.svg)](https://github.com/richtabor/coblocks/blob/master/license.txt)
+[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)](https://circleci.com/gh/godaddy-wordpress/coblocks) [![WordPress plugin](https://img.shields.io/wordpress/plugin/dt/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress plugin](https://img.shields.io/wordpress/plugin/v/coblocks.svg?style=flat)](https://wordpress.org/plugins/coblocks/) [![WordPress](https://img.shields.io/wordpress/v/coblocks.svg?style=flat)]() [![License](https://img.shields.io/badge/license-GPL--2.0%2B-red.svg)](https://github.com/godaddy-wordpress/coblocks/blob/master/LICENSE)
```
Comment (AnthonyLedesma) [inline README.md]: [CircleCI]
Should be
[CircleCI]

```
[![CircleCI](https://circleci.com/gh/godaddy/coblocks.svg?style=svg)]
Should be
[![CircleCI](https://circleci.com/gh/godaddy-wordpress/coblocks.svg?style=svg)]
```

File: package.json [hunk 2]
```diff
@@ -9,14 +9,14 @@
 	"repository": "godaddy/coblocks",
```
Comment (AnthonyLedesma) [inline package.json]: "godaddy/coblocks"
Should be
"godaddy-wordpress/coblocks"

**Comment by AnthonyLedesma** [comment]
Should we also update ./composer.json?
`"name": "godaddy/coblocks",`
to
`"name": "godaddy-wordpress/coblocks",`

File: README.md (hunk 1 above)
Comment (EvanHerman) [reply README.md]: @richtabor In my PR https://github.com/godaddy-wordpress/coblocks/pull/1162 (which I've closed) I used the following, which sets the default branch to master:
`[CircleCI](https://circleci.com/gh/godaddy-wordpress/coblocks/tree/master)`

Comment (richtabor) [reply README.md]: Thanks!

**Pull request merged**